test/test.js

```diff
@@ -72,8 +72,8 @@ test('strings', t=>{
         jams`{{bad key} val}`
     })
 
-    o = jams(`{key "val"}`)
-    t.equal(o.key, "val")
+    o = jams(String.raw`{"\"key" "\"val"}`) // Need raw so js doesn't escape \
+    t.equal(o[`"key`], `"val`)
```

Comment:
This test shouldn’t even pass based on latest requirement
The point is to separate out bareword part and at least get that done first 

Also global replace like this is probably never the right solution